group_vars/sn07.yml

```diff
@@ -431,6 +431,8 @@ galaxy_config_files:
     dest: "{{ galaxy_config['galaxy']['ucsc_build_sites'] }}"
   - src: "{{ galaxy_config_file_src_dir }}/config/echo_main_env.xml"
     dest: "{{ galaxy_config_dir }}/echo_main_env.xml"
+  - src: "{{ galaxy_config_file_src_dir }}/dynamic_rules/usegalaxy/total_perspective_vortex/tool_defaults.yml"
+    dest: "{{ tpv_mutable_dir }}/tool_defaults.yml"    
```

Comment:
```suggestion
    dest: "{{ tpv_mutable_dir }}/tool_defaults.yml"
```

Reply:
This should have been fixed in [9790734](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/689/commits/979073442aa293068a521fcc02e986aa5b88eaef) already.